api/code/fileManager/download/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../../../tsconfig.json",
```

Comment:
Some of the paths were simply wrong and they pointed to non-existing files. That's because the script for generating tsconfigs didn't take into account the depth of the project directory, and for api functions this depth varies. I fixed the script, so it works well.

Referencing directly `tsconfig.build.json` is a very important one, because otherwise, we have all the packages referenced by `compilerOptions.paths` configuration. This doesn't work, because it treats referenced projects' files as they belong to the current one, so for example `~/` in referenced project files is calculated relative to the current project, not the referenced one.

Basically, we had a lot of the wrong configurations and my IDE (vscode) was all red on some of `tsconfigs`. Babel seems to be very liberal in terms of how it treats this configuration, TS is not.